scripts/dockerfiles/run.sh

```diff
@@ -85,6 +90,58 @@ done
         /var/vcap/jobs/{{ $job.Name }}/bin/run
     {{ end }}
 {{ else }}
+    # Run all the scripts called post-start, if any.
+    psp=$(find /var/vcap/jobs/*/bin -name post-start)
+
+    if [ "X$psp" != X ] ; then
```

Comment:
I guess we could do that. It does offend my sense of aesthetics a bit to create something which does nothing.